apps/project/permissions.py

```diff
@@ -84,6 +84,15 @@ def has_object_permission(self, request, view, obj):
         )
 
 
+class IsProjectMember(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        from project.models import ProjectMembership
+        return ProjectMembership.objects.filter(
+            project=obj,
+            member=request.user
+        ).exists()
```

Comment:
This already exists.

Let's do this.
- Move `has_object_permission` logic there.
- Also use `project.members.filter` directly
- Make sure to check if `obj` is a project object.

https://github.com/the-deep/server/blob/17244762287d5b2fca2bdadd5017a5402730c602/deep/permissions.py#L156-L170